cookbooks/aws-parallelcluster-platform/metadata.rb

```diff
@@ -18,3 +18,5 @@
 depends 'selinux', '~> 6.0.5'
 
 depends 'aws-parallelcluster-shared', '~> 3.7.0'
+# TOFIX: remove this when all the common has been moved to shared
+depends 'aws-parallelcluster-common', '~> 3.7.0'
```

Comment:
The dependency on common cookbook should not be added.

You should rather try and move attributes and/or functions you need away from common cookbook, as close as you can to the code part using it. It might be the recipe or resource itself if it's the only code part using the attribute or the function, or the cookbook if the usage is limited to it. In (hopefully, rare) cases when multiple cookbooks use that code, it should be moved to the shared cookbook. In your case, `get_vpc_cidr_list` function is used or going to be used both by platform and environment code, so it can be moved to shared cookbook.

Reply:
Made changes w.r.t moving `get_vpc_cidr_list` to shared cookbook